kstreams-live-update/docker-compose.yaml

```diff
@@ -50,11 +55,12 @@ services:
     build:
       context: event-source
     links:
-     - mysql
+     - postgres
     environment:
+     - DATABASE_SERVER=postgres 
```

Comment:
In  case you want to go for one Compose file with both services, you'd have to keep both links and both depends_on. Then you can inject the server name like so:

    - DATABASE_SERVER=${DATABASE_SERVER:-mysql}

Then, to run with Postgres do `DATABASE_SERVER=postgres docker-compose --build up`.
    environment:
      KAFKA_BOOTSTRAP_SERVERS: kafka:9092

Reply:
Honestly I think it is simpler to have 2 compose files. That way we don't have one service running that really isn't necessary

Reply:
Done.